### DISCUSSION TITLE
adding tooltip for panel title

### DISCUSSION THREAD

**Discussion by yarix** [grafana/grafana#44076] [Dashboards, Panels & Visualizations]
in many cases where there are multiple panels on a row the title gets "..." after first few letters. this is expected.
i think that a tool tip during mouse hover will improve the usability of the panel.
in this example the panel title contains also the server name (on repeat) but you can't know which server you look at unless you remove some other panels or go to "view" mode.
a tool tip on mouse hover would be great here.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
